podpac/core/data/datasource.py

```diff
@@ -347,70 +394,19 @@ def _eval(self, coordinates, output=None):
         if requested_dims is not None and requested_dims != output.dims:
             output = output.transpose(*requested_dims)
         
+        # transform the output crs to the r]equested coordinates crs
+        # TODO: this could be made into method in/for UnitsDataArray 
```

Comment:
I think that's a good idea, but this is clean enough and you only do it one location, right?

Reply:
I will do this as part of the todo after #228